community/modules/scripts/ramble-access/main.tf

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  access_file = templatefile(
+    "${path.module}/templates/ramble_access.yml",
+    {
+      ramble_path = var.ramble_path
+    }
+  )
+
+  deps_file = templatefile(
+    "${path.module}/templates/install_ramble_deps.yml",
```

Comment:
nit: totally an opinion here: I would move this block up in the file so install deps is before access as that is the order they actually run in.